src/SprykerSdk/Zed/SprykGui/SprykGuiConfig.php

```diff
@@ -28,4 +28,12 @@ public function getProjectNamespaces(): array
     {
         return Config::get(KernelConstants::PROJECT_NAMESPACES, []);
     }
+
+    /**
+     * @return bool
+     */
+    public function isSprykGuiEnabled(): bool
+    {
+        return APPLICATION_ENV === 'production';
```

Comment:
That is actually the opposite - on production it must NOT be enabled for security reasons.
And this must not be overwritable.